pkg/controller/kieapp/webconsole.go

```diff
@@ -0,0 +1,93 @@
+package kieapp
+
+//go:generate go run -mod=vendor ./defaults/.packr/packr.go
+
+import (
+	"context"
+	"errors"
+	"fmt"
+	"github.com/RHsyseng/operator-utils/pkg/utils/openshift"
+	"github.com/ghodss/yaml"
+	"github.com/gobuffalo/packr/v2"
+	api "github.com/kiegroup/kie-cloud-operator/pkg/apis/app/v2"
+)
+
+func CreateConsoleYAMLSamples(r *Reconciler) {
```

Comment:
I think this file will look much better if you combine these 4 into one function.

Reply:
modified